onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -465,12 +465,6 @@ def _create_inference_session(self, providers, provider_options, disabled_optimi
         )
         if not providers and len(available_providers) > 1:
             self.disable_fallback()
```

Comment:
We don't need to call this. In fact, the we don't need the 'if' block at all.

Reply:
Let's drop it then.